src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup.mdx

```diff
@@ -125,21 +136,35 @@ Before you go to the external OTLP exporter documentation, consult the table bel
 In Node.js, the [opentelemetry-collector-exporter-grpc](https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-grpc) library requires additional options to enable TLS.
```

Comment:
in the table above, the last endpoint is in code format (unclickable) but the others are clickable. Not sure but maybe they shouldn't be clickable?

Reply:
@zuluecho9, I removed the link markup. Great suggestion. I'll stick with the UI links for license key.